index.js

```diff
@@ -1,15 +1,15 @@
-const { createProbot } = require('probot')
-const { resolve } = require('probot/lib/resolver')
-const { findPrivateKey } = require('probot/lib/private-key')
+const { Probot } = require('probot')
+const { resolve } = require('probot/lib/helpers/resolve-app-function')
+const { findPrivateKey } = require('probot/lib/helpers/get-private-key')
```

Comment:
I'm not happy that we use internal APIs, I'll see if we can make these methods official exports at the root level so things won't break if Probot's internal file structure changes.